apps/system/utils/README.md

```diff
@@ -140,27 +144,19 @@ Options:
  -i PRINT_INTERVAL     Show cpuload values every PRINT_INTERVAL(s)
  -n ITERATION_COUNT    Iterate showing cpuload values ITERATION_COUNT times
 
-TASH>> cpuload
-Started CPU monitor with interval 5.
-PID | Pri |    2s |
+TASH>>cpuload
+
+--------------------------------------------------
+Started CPU monitor with interval 5 sec.
+TASH>>Non-Zero CPU utilization (2s)
+PID | Pri |  CPU0 |  CPU1 | 2s Avg|
```

Comment:
We dont have to mention 2s Avg here. All the values (each cpu value) are averaged over 2s.